cart-api-service/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:12-alpine
+
+# Create app directory inside the image
+WORKDIR /usr/src/app
+
+# Copy package.json and package-lock.json file from the current directory to the image
+COPY package*.json ./
+
+# Install node_modules
+RUN npm i && npm cache clean --force
```

Comment:
You're including dev dependencies by this way

Reply:
Added the flag for production